### ISSUE TITLE
New rule: SingleLinePerProperty

### ISSUE THREAD

**Issue by nschonni** [brigade/scss-lint#120]
Similar to `SingleLinePerSelector`, it would require each property to be on new lines. See comments inside code block for possible blockers
**Bad**

``` scss
// This style is preferred by some styleguides (Bootstrap) when only a single property is set
// This might required a relaxing option, or be accepted by default
.foo { color: red; }

//Don't think anyone recommends multiple properties without a newline ...
.bar {
  color: red; background: blue;
}
```

**Good**

``` scss
.foo {
  color: red:
}

.bar {
  color: red;
  background: blue;
}
```
